pom.xml

```diff
@@ -19,19 +19,26 @@
             <artifactId>selenium-java</artifactId>
             <version>4.9.1</version>
         </dependency>
-
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
             <version>7.8.0</version>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>io.github.bonigarcia</groupId>
             <artifactId>webdrivermanager</artifactId>
             <version>5.3.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.7</version>
+        </dependency>
```

Comment:
slf4j-api is useless here, it does nothing.

Reply:
after deleting it does this:

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

Reply:
Re-build project manually before execution (Ctrl+F9 in IntelliJ) – it will compile new libraries into “target”. Otherwise, it uses previously builded libraries.

Reply:
after rebuilding issue is still there :(